rbs/sa.tf

```diff
@@ -30,6 +30,12 @@ locals {
     "roles/resourcemanager.projectCreator",
     "roles/resourcemanager.projectDeleter",
   ]
+
+  folder_role_to_folder_id_map = [
```

Comment:
is it a map or a list of objects?
nit: how about `folder_ids_and_roles`? or `folder_iam_bindings`?